Xamarin.Forms.Xaml/MarkupExtensionParser.cs

```diff
@@ -24,6 +24,8 @@ public object Parse(string match, ref string remaining, IServiceProvider service
 				markupExtension = new OnIdiomExtension();
 			else if (match == "DataTemplate")
 				markupExtension = new DataTemplateExtension();
+			else if (match == "FontImage")
```

Comment:
not required. I'm not even sure we save any measurable time doing this

Reply:
I can remove this, no problem